uniffi_bindgen/src/interface/object.rs

```diff
@@ -128,6 +128,14 @@ impl Object {
         self.uses_deprecated_threadsafe_attribute
     }
 
+    pub fn iter_ffi_function_definitions(&self) -> Vec<FFIFunction> {
+        vec![self.ffi_object_free().clone()]
+            .into_iter()
+            .chain(self.constructors.iter().map(|f| f.ffi_func.clone()))
+            .chain(self.methods.iter().map(|f| f.ffi_func.clone()))
+            .collect()
+    }
```

Comment:
We should like to be able to get `CodeDeclaration` to dispense their own `FFIFunction`s, but this isn't done yet.